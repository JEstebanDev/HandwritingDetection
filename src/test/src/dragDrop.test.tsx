import { render, screen } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import DragDrop from "../../src/components/dragDrop";
describe("working propertly", () => {
  test("should render the DragDrop", () => {
    const { container } = render(
      <MemoryRouter initialEntries={["/"]}>
        <DragDrop />
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
