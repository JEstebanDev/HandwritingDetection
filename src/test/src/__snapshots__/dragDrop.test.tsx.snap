// Vitest Snapshot v1

exports[`working propertly > should render the DragDrop 1`] = `
<div>
  <div
    class="FilesDragAndDrop"
  />
</div>
`;
